ember/config/environment.js

```diff
@@ -179,6 +178,9 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV["ember-simple-auth-oidc"].host =
+      process.env.OIDC_HOST ||
+      "https://mysagw.local/auth/realms/mysagw/protocol/openid-connect";
```

Comment:
Is this even needed? In dev it all runs under `mysagw.local`. Or would it not work on a different port (i.e. `4200`)?

Reply:
Yes, when using a port for the ember dev server it does not work.